MyTests/Tests/UnitTest1.cs

```diff
@@ -0,0 +1,19 @@
+using MyTests.Tests;
```

Comment:
pls check your .cproj file as seems it thinks that you have two UnitTest1.cs files